css/properties.json

```diff
@@ -5833,6 +5833,21 @@
     "order": "lengthOrPercentageBeforeKeywords",
     "status": "standard"
   },
+  "text-justify": {
+    "syntax": "auto | distribute | inter-character | inter-word | none",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Text"
+    ],
+    "initial": "auto",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
```

Comment:
I put it as uniqueOrder, because the newest spec says "n/a", but I don't know if this is right. I honestly don't know this stuff very well. I'm going to suggest that we review this syntax in next weeks' editors meeting